hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/TestParentQueue.java

```diff
@@ -131,10 +131,10 @@ private void setupSingleLevelQueuesWithAbsoluteResource(
     // Define top-level queues
     conf.setQueues(CapacitySchedulerConfiguration.ROOT, new String[]{A, B});
 
-    conf.setMinimumResourceRequirement("", Q_A,
```

Comment:
I think these fields should be QueuePath objects:
```
  private static final String Q_A =
      CapacitySchedulerConfiguration.ROOT + "." + A;
  private static final String Q_B =
      CapacitySchedulerConfiguration.ROOT + "." + B;
```

Reply:
Q_A as a QueuePath object is used only here. Same with  Q_B. These variables are used in setCapacity(), setQueues() and set() methods.

Reply:
To have a QueuePath from Q_A and Q_B are only in this method is necessary. So the only occurrence for this purpose is only one.
Q_A and Q_B are used as string in setCapacity(), setQueues() and set() methods.